client/landing/subscriptions/components/add-sites-button/add-sites-button.tsx

```diff
@@ -13,23 +13,20 @@ const AddSitesButton = () => {
 	const translate = useTranslate();
 	const [ isAddSitesModalVisible, setIsAddSitesModalVisible ] = useState( false );
 	const dispatch = useDispatch();
-	const resendEmailVerification = useResendEmailVerification();
 	const isEmailVerified = useSelector( isCurrentUserEmailVerified );
 
 	return (
 		<>
 			<Button
-				primary
-				className="subscriptions-add-sites__button"
+				variant="primary"
+				className="button subscriptions-add-sites__button"
 				onClick={ () => {
 					if ( ! isEmailVerified ) {
 						return dispatch(
-							errorNotice( translate( 'Your email has not been verified yet.' ), {
```

Comment:
This same messaging exists in some other places as well, any particular reason of not change all instances?

Reply:
Nope. It may be worth updating those too, depending on their context. 🤔

Reply:
Actually I am working on those other messages in https://github.com/Automattic/wp-calypso/issues/98434 so please do let me know if they need any update.

Personally I think it's fine to have those.